src/buildercore/fastly/vcl/journal-submit.vcl

```diff
@@ -0,0 +1,12 @@
+if (req.url.path == "/submit") {
+    if (std.prefixof(req.http.Referer, "https://staging--xpub.elifesciences.org/")) {
```

Comment:
currently xpub redirects to `continuumtest--journal.elifesciences.org` so this needs to be changed to the CDN URL for proper testing